button/button.css

```diff
@@ -0,0 +1,191 @@
+@import url(/style.css);
+
+.btn-intro {
+  font-size: 1.25rem;
+  padding: 1rem 0;
+}
+.btn-container {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  gap: 2rem;
+  height: 16rem;
+  margin-bottom: 1rem;
+}
+
+/* primary btns */
+
+.primary-btn-sm,
+.primary-btn-md,
+.primary-btn-lg,
+.btn-with-icon-sm,
+.btn-with-icon-md,
+.btn-with-icon-lg {
+  border: none;
+  color: var(--dark-black);
+  border: none;
+  font-size: 1.25rem;
+}
```

Comment:
can define a separate class for this styling

Reply:
I got reviewed this from mentor, she told me its good practice.

Reply:
alright